dnd5e.mjs

```diff
@@ -25,6 +25,7 @@ import {default as registry} from "./module/registry.mjs";
 import * as utils from "./module/utils.mjs";
 import {ModuleArt} from "./module/module-art.mjs";
 import registerModuleData from "./module/module-registration.mjs";
+import Sockets5e from "./module/sockets.mjs";
```

Comment:
It would be nice to have the socket orchestration class inside the `sockets` folder, rather than having two entries named `sockets` inside the module folder.